client/src/main/scala/drt/client/services/JSDateConversions.scala

```diff
@@ -109,7 +110,10 @@ object JSDateConversions {
     }
 
     def stringToSDateLikeOption(dateString: String): Option[SDateLike] = {
-      Try(JSSDate(Moment(dateString).toDate())).toOption
+      Try{
```

Comment:
I think it would be neater to write this as
```
if(moment.isValid())
  Option(JSSDate(moment.toDate()))
else None
```
rather than throwing an Exception to trigger the None case.